### ISSUE TITLE
How do i fix this

### ISSUE THREAD

**Issue by 023948799e213e21b53eb83785356208** [totoroterror/warp-cloner#26]
Failed to register: 400 {"success":false,"result":null,"errors":["Invalid registration request"],"messages":[]} (key: Failed to register: 400 {"success":false,"result":null,"errors":["Invalid registration request"],"messages":[]} (key: XXXXX-XXXXX-XXXXX)

It always shows even tho i have fully working keys any ways how to fix
(i redacted the key so no one connects)

**Comment by almaftuchin** [comment]
me too get this error please @totoroterror fix this issue

**Comment by totoroterror** [comment]
Seems like Cloudflare have blocked requests with partial body. This is fixable, I'll fix this later this week, when I'll be at home. If somebody want to do this faster – PR are open.

**Comment by venscn** [comment]
The body need to add key field. @totoroterror 

<img width="871" alt="image" src="https://github.com/totoroterror/warp-cloner/assets/11350982/ab2d41e6-2447-4354-8b25-eef961c68b81">

**Comment by totoroterror** [comment]
> The body need to add key field. @totoroterror I don’t have a PB key available. Can you generate a private message for me?
> 
> <img alt="image" width="871" src="https://private-user-images.githubusercontent.com/11350982/287656435-ab2d41e6-2447-4354-8b25-eef961c68b81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE2ODQ3NjksIm5iZiI6MTcwMTY4NDQ2OSwicGF0aCI6Ii8xMTM1MDk4Mi8yODc2NTY0MzUtYWIyZDQxZTYtMjQ0Ny00MzU0LThiMjUtZWVmOTYxYzY4YjgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA0VDEwMDc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQyNzc2NDllZGIwZWJlZWQ3Mjk4M2QxM2M0N2FhNjFmZjk5MzJmMmFmOWU2ZDI0ZmE1NzNlNzViMDZiNjEzODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.NMmXcEvzM40sWIXp_lWihRMiDO7qDNtGLMkKfbtAE3k">

No, it's still working fine. Just need some updates. My friend already fixed this, I'll release update when I'll be at home

**Comment by 023948799e213e21b53eb83785356208** [comment]
> The body need to add key field. @totoroterror I don’t have a PB key available. Can you generate a private message for me?
> 
> <img alt="image" width="871" src="https://private-user-images.githubusercontent.com/11350982/287656435-ab2d41e6-2447-4354-8b25-eef961c68b81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NTYyNzMsIm5iZiI6MTcwMTc1NTk3MywicGF0aCI6Ii8xMTM1MDk4Mi8yODc2NTY0MzUtYWIyZDQxZTYtMjQ0Ny00MzU0LThiMjUtZWVmOTYxYzY4YjgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDA1NTkzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxMDcwOTI0MDE5YzhhOTU5ZTY1MDBiNWZhNzFiOTkwNzZkMzRjNjNiMWIxYmZlYmVlZmFhMzY5MzBmYzhlYmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rRhiVvLfgsKV5WCvtlvQ4w5RYY5ijHzEVQ1MGthbNdY">

U can add me on discord i have some keys:
runnymanny8481

**Comment by 023948799e213e21b53eb83785356208** [comment]
> @023948799e213e21b53eb83785356208 Thanks a lot, I've got it in telegram and cloned some using the current project.

Also do u have a fixed version of warp-cloner? if so release it pls

**Comment by ShatteredDisk** [comment]
Agreed, it would be much appreciated if you could get it to work again

**Comment by ShatteredDisk** [comment]
Thanks! I’ll test it tonight when I get a chance

**Comment by ShatteredDisk** [comment]
@venscn @totoroterror created pull request (#27)

**Comment by totoroterror** [comment]
fixed in b4366ad3d1d54ae3d6df386dd3894097676cc403
